prow/config.yaml

```diff
@@ -12300,7 +12816,9 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+
 - interval: 6h
+  agent: kubernetes
```

Comment:
It was easier to append with sed by using interval as opposed to name, since name is also found inside the container spec. I wouldn't expect template order to be an issue, during marshaling everything should fall into the right place because of the struct tags, right?